.gitignore

```diff
@@ -26,3 +26,4 @@ osdctl
 # goReleaser 
 dist/
 bin/
+vendor/
```

Comment:
Should we be ignoring `vendor`?

Reply:
Apologies, that was a mistake on my end. I’ll remove the changes .